clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -385,6 +385,9 @@ DPCT_ENUM_OPTION(DPCT_OPT_TYPE(static llvm::cl::bits<ExperimentalFeatures>), Exp
         false),
              DPCT_OPT_ENUM("occupancy-calculation", int(ExperimentalFeatures::Exp_OccupancyCalculation),
         "Experimental helper function used to calculate occupancy.\n",
+        false),
+             DPCT_OPT_ENUM("oneapi-joint-matrix", int(ExperimentalFeatures::Exp_JointMatrix),
```

Comment:
https://github.com/intel/llvm/tree/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix 
the extension name is: sycl_ext_oneapi_matrix,  so let's use the name: matrix instead of oneapi-joint-matrix